openedx/core/djangoapps/content_libraries/models.py

```diff
@@ -71,6 +118,15 @@ class ContentLibrary(models.Model):
         """),
     )
 
+    lti_tool = models.ForeignKey(
+        LtiTool,
+        default=None,
+        null=True,
+        on_delete=models.SET_NULL,
+        help_text=("Authorize the  LTI tool selected to expose this library's content "
+                   "through LTI launches, leave unselected to disable LTI launches.")
```

Comment:
Correct my if I'm wrong but isn't this about authorizing an LTI _Platform_ to access this content library? (This content library is the tool). I think the [`LtiTool`](https://github.com/dmitry-viskov/pylti1.3/blob/ec931d3117889b7236048b3426fc9af0321b7bc6/pylti1p3/contrib/django/lti1p3_tool_config/models.py#L37) model is poorly named, as it seems like it holds various data values about a specific _platform_, and is used by a tool to authenticate the platform.

Docstring aside, does this setup allow multiple LTI Platforms (multiple external LMSs) to use a single content library as an LTI tool? From my superficial understanding of this it seems like this will allow multiple content libraries (tools) to be authorized to the same external LMS (platforms), which is good, but that it would not allow a single content library to be used by multiple external LMSs, which definitely needs to be supported as that's one of the main reasons for centralizing content into a shared library.